config/locales/he.yml

```diff
@@ -605,6 +608,7 @@ he:
         projectmgmt_is_dream_near_theme_camp: :form_project_management_is_dream_near_theme_camp
         projectmgmt_dream_pre_construction_site: :form_project_management_dream_pre_construction_site
         dream_point_of_contact_email: :form_dream_point_of_contact_email
+        safety_file_comments: :dream_safety_file_comments_title
```

Comment:
Generally, I'd try to use aliases only if there's no other way, as it makes translation work much harder (the aliases will show twice in the to-be-translated file). 
Maybe use "activerecord.attributes.camp.safety_file_comments" instead? 
You can also use it as a label of the field within the form and let SimpleForm do that automatically for you (`f.label :safety_file_comments`)